src/commons/grading/GradingFlex.tsx

```diff
@@ -0,0 +1,56 @@
+declare const twJustifyContentValues: readonly ["justify-start", "justify-end", "justify-center", "justify-between", "justify-around", "justify-evenly"];
+declare type JustifyContent = typeof twJustifyContentValues[number];
+declare const twAlignItemsValues: readonly ["items-start", "items-end", "items-center", "items-baseline", "items-stretch"];
+declare type AlignItems = typeof twAlignItemsValues[number];
+declare const twFlexDirectionValues: readonly ["row", "column"];
+declare type FlexDirection = typeof twFlexDirectionValues[number];
+
+type GradingFlexProps = {
+    justifyContent?: JustifyContent;
+    alignItems?: AlignItems;
+    flexDirection?: FlexDirection;
+    children?: React.ReactNode;
+    style?: React.CSSProperties;
+    className?: string;
+} & React.RefAttributes<HTMLDivElement>;
+
+const GradingFlex: React.FC<GradingFlexProps> = ({ justifyContent, alignItems, flexDirection, children, style, className, }: GradingFlexProps) => {
+  const defaultStyle: React.CSSProperties = {
+    display: "flex",
+    justifyContent: 
+      (justifyContent === "justify-start"
+        ? "flex-start"
+        : justifyContent === "justify-end"
+        ? "flex-end"
+        : justifyContent === "justify-center"
+        ? "center"
+        : justifyContent === "justify-between"
+        ? "space-between"
+        : justifyContent === "justify-around"
+        ? "space-around"
+        : justifyContent === "justify-evenly"
+        ? "space-evenly"
+        : ""
+      ),
```

Comment:
Seems like we are mapping Tailwind values to actual styles – what's the need for this? Can we just map input props directly?

Reply:
I was just being lazy and trying to avoid touching existing props but I can go through all of it again when I work on it to remove it.

Reply:
This has been fixed in the latest commit, it can be resolved once you are ok with it.